test/verify/check-packagekit

```diff
@@ -1021,11 +1021,12 @@ class TestAutoUpdates(PackageCase):
         self.assertNotIn("vanilla", out)
         self.assertNotIn("Shutdown", out)
 
+    @skipImage("No supported auto update backend", "centos-7", "debian-stable", "debian-testing", "ubuntu-1604", "ubuntu-stable", "rhel-7-5", "rhel-7-5-distropkg", "rhel-7-6")
```

Comment:
The point of this test is to see what happens on OSes that don't have a supported auto update backend. So it should not be skipped there.